index.html

```diff
@@ -20,11 +20,105 @@
               <li class="web_buttons_2"><a href="https://linkedin.com/in/sumair-qaisar-jadoon-84a877164"><img src="assets/images/social_media/linkedin.svg" alt="Linkedin icon"></a></li>
               <li class="web_buttons_3"><a href="https://github.com/sumairq"><img src="assets/images/social_media/github.svg" alt="Github icon"></a></li>
             </ul>
+            <a id="down_icon" href="#"><img src="assets/images/down.png" alt="Icon for moving"></a>
       </section>
+      <header>
+        <h2 class="text box_1">My Recent Works </h2>
+        <img class="center" src="assets/images/Indicator.png" alt="Design line">
+
+      </header>
+
+      <!-- main grid container -->
+      <section class="grid_container">   
+        <!-- flexbox container   -->
+        <div class="flex_container_primary">
+          <img src="./assets/images/img-placeholder.png" alt="placeholder for project image">
+          <div class="overlap"><h3>Multi-Post Stories<br>Gain + Glory</h3>
+                <ul>
+                  <li>Ruby on rails</li>
+                  <li>CSS</li>
+                  <li>JavaScript</li>
+                  <li>HTML</li>
+                </ul>
+                <a class="web_button" href="#">See Project</a>
+                </div>
+          </div>
+          <div class="flex_container_primary">
+            <img src="./assets/images/img-placeholder.png" alt="placeholder for project image">
+            <div class="overlap"><h3>Multi-Post Stories<br>Gain + Glory</h3>
+                  <ul>
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                  </ul>
+                  <a class="web_button" href="#">See Project</a>
+                  </div>
+            </div>
+            <div class="flex_container_primary">
+              <img src="./assets/images/img-placeholder.png" alt="placeholder for project image">
+              <div class="overlap"><h3>Multi-Post Stories<br>Gain + Glory</h3>
+                    <ul>
+                      <li>Ruby on rails</li>
+                      <li>CSS</li>
+                      <li>JavaScript</li>
+                      <li>HTML</li>
+                    </ul>
+                    <a class="web_button" href="#">See Project</a>
+                    </div>
+              </div>
+      
+      
+        
+      </section>
+
+      <header class="background_aboutme">
+        <h2 class="text box_1">About me</h2>
+        <section class="grid_container">
+        <h3 class="aboutme_text">I can help you build a product ,<br> feature or website. Look through<br> some of my work and experience!<br> If you like what you see and have<br> a project you need coded,<br> don’t hesitate to contact me.</h3>
+        <a class="web_button center-border" href="#">Get My Resume</a>
+        </section>
+      </header>
```

Comment:
- [ ] To keep your HTML semantic, you should have this header inside the upcoming section. This whole section should be about you, so it's logical to have a little introduction about yourself and then your skills all in the same place.
- [ ] You should consider making the `<h3 class="aboutme_text">` a `<p class="aboutme_text">`.